vowpalwabbit/cb_explore_adf_common.h

```diff
@@ -152,6 +152,18 @@ inline void cb_explore_adf_base<ExploreType>::learn(
     data.explore.learn(base, examples);
     if (data._metrics)
     {
+      size_t num_features = 0;
+      size_t num_namespaces = 0;
+
+      for (const auto& example : examples)
+      {
+        num_features += example->get_num_features();
```

Comment:
@jackgerrits get_num_features can be called only after learn right?

Reply:
Yes, the interacted count requires learn be called

Reply:
oh so should be moved after learn